src/mp_api/client.py

```diff
@@ -390,12 +401,293 @@ def get_entries(
+            - {Element("H"), Element("O")}
+        )
+        for entry in ion_ref_entries:
+            entry_elts = set(entry.composition.elements)
+            # Ensure no OH chemsys or extraneous elements from ion references
+            if not (
+                entry_elts <= {Element("H"), Element("O")}
+                or extra_elts.intersection(entry_elts)
+            ):
+                # Create new computed entry
+                form_e = ion_ref_pd.get_form_energy(entry)
+                new_entry = ComputedEntry(
+                    entry.composition, form_e, entry_id=entry.entry_id
+                )
+                pbx_entry = PourbaixEntry(new_entry)
+                pbx_entries.append(pbx_entry)
+
+        return pbx_entries
+
+    # TODO - @lru_cache causes this method to fail when chemsys is given as a list,
+    # with an 'unhashable type' error
+    # @lru_cache
+    def get_ion_reference_data(self, chemsys: Union[str, List]) -> List[dict]:
+        """
+        Download aqueous ion reference data used in the construction of Pourbaix Diagrams.
+
+        Use this method to examine the ion data and to add additional
+        ions if desired. The data returned from this method can be passed to
+        get_ion_entries().
+
+        Data are retrieved from the Aqueous Ion Reference Data project
+        hosted on MPContribs. Refer to that project and its associated documentation
+        for more details about the format and meaning of the data.
+
+        Args:
+            chemsys (str or [str]): Chemical system string comprising element
+                symbols separated by dashes, e.g., "Li-Fe-O" or List of element
+                symbols, e.g., ["Li", "Fe", "O"].
+
+        Returns:
+            [dict]: Among other data, each record contains 1) the experimental ion  free energy, 2) the
+                formula of the reference solid for the ion, and 3) the experimental free energy of the
+                reference solid. All energies are given in kJ/mol. An example is given below.
+
+                {'identifier': 'Li[+]',
+                'formula': 'Li[+]',
+                'data': {'charge': {'display': '1.0', 'value': 1.0, 'unit': ''},
+                'ΔGᶠ': {'display': '-293.71 kJ/mol', 'value': -293.71, 'unit': 'kJ/mol'},
+                'MajElements': 'Li',
+                'RefSolid': 'Li2O',
+                'ΔGᶠRefSolid': {'display': '-561.2 kJ/mol',
+                    'value': -561.2,
+                    'unit': 'kJ/mol'},
+                'reference': 'H. E. Barner and R. V. Scheuerman, Handbook of thermochemical data for
+                compounds and aqueous species, Wiley, New York (1978)'}}
+        """
+        if isinstance(chemsys, str):
+            chemsys = chemsys.split("-")
+        # capitalize and sort the elements
+        chemsys = sorted(e.capitalize() for e in chemsys)
+
+        # TODO - see if there is a way to avoid querying the entire collection
```

Comment:
Yes. You can query for a formula containing a substring with `formula__contains` or any of the data fields, see https://contribs-api.materialsproject.org/#/contributions/get_entries for syntax.

Reply:
OK. What we would really need here is the equivalent of mongo `$in`, i.e. {'MajElements`: {"$in": chemsys}}. As best I can tell that isn't possible.

Reply:
I refer that to @tschaume :)

Reply:
You should be able to use `data__MajElements__in`, e.g. https://contribs-api.materialsproject.org/contributions/?project=ion_ref_data&data__MajElements__in=Li,B&_fields=identifier,data.MajElements